### ISSUE TITLE
Can't see inventory/guild bank/own bank

### ISSUE THREAD

**Issue by Paletress** [tullamods/Bagnon#891]
**Running Software (please complete the following information):**
 - Game patch [e.g. Build 8.0.1.27178] Latest
 - Addon version 8.1.0

**Describe the bug**
I can't see my inventory bags,  my guild bank slot and my own bank slots, It appear only a small string with the commands. On another character the inventory works, but the banks nope. Is it something you're going to fix?  
**To Reproduce**

**Expected behavior**

**Screenshots**

**Error Logs**
If applicable, add the first error message (labeled 1/Many) that appeared.

**Additional context**

**Comment by AvalynnJust** [comment]
I have a similar problem that just started with the latest patch for WOW... I am running Bagnon 8.1.6
On one of my characters when I open her bags I just see the bag bar and I can not see any inventory, and the same thing happens if I go to the bank or the guild vault. I tried another character and it was just the bank and then another it was everything again so I tried to uninstall and re-install but that did not help.  So for now I just disabled it.  I am hope I can either figure out what I did wrong or maybe an update will fix it.  There is no error log I just can't see what is in my bags they won't open.
